support-frontend/assets/pages/digital-subscription-checkout/components/digitalCheckoutFormGift.jsx

```diff
@@ -225,23 +223,14 @@ function DigitalCheckoutFormGift(props: PropTypes) {
           <FormSection title="Billing address">
             <Address />
           </FormSection>
-          <PaymentMethodSelector
-            country={props.country}
-            paymentMethod={props.paymentMethod}
-            setPaymentMethod={props.setPaymentMethod}
-            onPaymentAuthorised={props.onPaymentAuthorised}
-            validationError={firstError('paymentMethod', props.formErrors)}
-            csrf={props.csrf}
-            currencyId={props.currencyId}
-            payPalHasLoaded={props.payPalHasLoaded}
-            formIsValid={props.formIsValid}
-            validateForm={props.validateForm}
-            isTestUser={props.isTestUser}
-            setupRecurringPayPalPayment={props.setupRecurringPayPalPayment}
-            amount={props.amount}
-            billingPeriod={props.billingPeriod}
-            allErrors={[...props.formErrors, ...props.addressErrors]}
-          />
+          <FormSection title="How would you like to pay?">
```

Comment:
Don't we need the `paymentMethods.length > 1 ?` check here?

Reply:
Yes, you're totally right. Thanks for picking this up!